fintrack/fintrack/settings.py

```diff
@@ -144,3 +146,22 @@
         'rest_framework.permissions.IsAuthenticated',
     ],
 }
+
+
+EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
+EMAIL_HOST = config('EMAIL_HOST')
+EMAIL_PORT = config('EMAIL_PORT')
+EMAIL_USE_TLS = config('EMAIL_USE_TLS')
+EMAIL_HOST_USER = config('EMAIL_HOST_USER')
+EMAIL_HOST_PASSWORD = config('EMAIL_HOST_PASSWORD')
+
+CELERY_BROKER_URL = 'redis://localhost:6379/0'
+
+CELERY_BEAT_SCHEDULER = 'django_celery_beat.schedulers:DatabaseScheduler'
+CELERY_TIMEZONE = 'Asia/Karachi'
+CELERY_BEAT_SCHEDULE = {
+    'send_transaction_emails': {
+        'task': 'transactions.tasks.send_transaction_emails',
```

Comment:
I would suggest to use django admin for scheduling task, that way upon change, deployment wouldn't have to done